CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Crash when copying/pasting with neither pointer nor keyboard focus on Wayland
 - Crash due to fd leak on Wayland
 - IME window position with fullwidth characters in the search bar
+- Selection sometimes expanding over 2 chars when scrolled up in history
```

Comment:
```suggestion
- Selection expanding over 2 characters when scrolled in history with fullwidth characters in use
```

I don't like being so vague here, since we know exactly what the issue is.

Reply:
I thought about it, but it was really too long, but if it's fine I'll use it.

Reply:
This is still under the 100 col limit we usually have for changelog entries?